lldb/tools/lldb-dap/DAP.cpp

```diff
@@ -1105,6 +1148,15 @@ void DAP::ConfigureSourceMaps() {
   RunLLDBCommands("Setting source map:", {sourceMapCommand});
 }
 
+// REMOVED: DetectNetworkSymbolServices - no longer needed due to core fixes
```

Comment:
We don't record features that were removed in the sources.  That gets recorded in the revision history.